.bazelrc

```diff
@@ -68,8 +67,10 @@ build:target-linux --host_cpu=k8
 #   bazel test --config=x //...
 build:remote-target-linux --config=target-linux
 build:remote-target-linux --config=remote
+build:remote-target-linux --experimental_platform_in_output_dir
 build:remote-target-linux-dev --config=target-linux
 build:remote-target-linux-dev --config=remote-dev
+build:remote-target-linux-dev --experimental_platform_in_output_dir
```

Comment:
would it make sense to put this in `build:target-linux`?

Reply:
I would be better to put it in "build" as default. But I was trying to be most conservative here.

Could move to target-linux as well.